spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestPartitionValues.java

```diff
@@ -432,4 +436,47 @@ public void testPartitionedByNestedString() throws Exception {
 
     Assert.assertEquals("Number of rows should match", rows.size(), actual.size());
   }
+
+  @Test
+  public void testReadPartitionColumn() throws Exception {
+    Assume.assumeTrue("Temporary skip ORC", !"orc".equals(format));
```

Comment:
Same as here too, temporary skip ORC test because https://github.com/apache/iceberg/pull/4599 is working on fixing it.